src/main.c

```diff
@@ -505,6 +505,88 @@ np2srv_sm_oper_cb(sr_session_ctx_t *session, uint32_t UNUSED(sub_id), const char
     return rc;
 }
 
+#ifdef NC_ENABLED_SSH_TLS
+
+void
+np2srv_cert_exp_notif_cb(const char *expiration_time, const char *xpath, void *user_data)
```

Comment:
Should be `static` and missing doxygen then (for callbacks with typedefs only a short brief description should be supplied).

Reply:
Added both.